cf_remote/remote.py

```diff
@@ -515,6 +515,12 @@ def install_host(
         return 1
 
     if remote_download:
+        if data.get("bin") and not data.get("bin").get("curl"):
```

Comment:
`dict.get()` returns None by default if the key is not found, so if you try to run `.get()` on that, you get `AttributeError`.

You can give an empty dict as a default value, so the last `.get()` also works:

```suggestion
        if data.get("bin") and not data.get("bin", {}).get("curl"):
```